pgo/cmd/upgrade.go

```diff
@@ -64,6 +69,7 @@ func init() {
 
 	// flags for "pgo upgrade"
 	UpgradeCmd.Flags().BoolVarP(&IgnoreValidation, "ignore-validation", "", false, "Disables version checking against the image tags when performing an cluster upgrade.")
+	UpgradeCmd.Flags().StringVarP(&PostGISImageTag, "post-gis-image-tag", "", "", "The PostGISImageTag to use for cluster creation. If specified, it overrides the default configuration setting.")
```

Comment:
Hm, I'm not totally sold on this, given this would be the first time we're specifically saying "hey, here's something special just for PostGIS!". Maybe it's something like `--ccp-image-tag` (which is also a name I'm not thrilled with but we have precedent).

If we do keep this flag, it would suggest `--postgis-image-tag`

Reply:
Yeah, this was something I went back and forth on. It does behave the same as the `--ccp-image-tag`, but was only added for PostGIS. The original upgrade design was to maintain, as much as possible, the cluster definition as defined in the pgcluster CRD and the default configuration. However, given the underlying functionality is the same, I'm fine with changing it to the more consistent name.
Updated, thanks!